### ISSUE TITLE
[Request] Hide unsupported file types

### ISSUE THREAD

**Issue by GoogleCodeExporter** [AmatCoder/mednaffe#13]
```
It would be a convenient option to be able to hide unsupported file types 
(.txt, .doc, .etc...) in the rom browser, so that only playable rom files are 
displayed.
```

Original issue reported on code.google.com by `StarrySk...@gmail.com` on 2 Feb 2014 at 10:02

**Comment by doyousketch2** [comment]
Yeah, this is a good idea.

Mednafen doesn't seem to look at file extensions when determining filetypes.
It just compares headers, and looks at the way the raw data is mapped out.

So there's no internal list of filetypes that I'm aware of.

You'd basically just have to go through all the various roms that medafen loads,
then unzip 'em all, so you can see the actual file extensions being used.

**Comment by AmatCoder** [comment]
Implemented on 18905b6 commit. (Well, kind of...)
Unsupported filetypes are not hidden but now you can choose that filetypes are shown on a per-folder basis.
